tests/publiccloud/pcw_target.pm

```diff
@@ -0,0 +1,39 @@
+use base "consoletest";
+use serial_terminal 'select_serial_terminal';
+use transactional qw(process_reboot);
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+
+sub run {
+    my ($self, $args) = @_;
+    select_serial_terminal;
+    record_info('system', script_output('cat /etc/os-release'));
+    record_info('device', script_output('nmcli -t device'));
+
+    assert_script_run('nmcli con add type ethernet con-name "static-ip" ifname ens4 ipv4.method manual ipv4.addresses 10.0.2.20/15 gw4 10.0.2.2');
+    assert_script_run('nmcli con mod static-ip ipv4.dns "10.136.53.53,10.136.53.54,10.100.2.10"');
```

Comment:
There is `mm_network::configure_static_ip`, `mm_network::configure_default_gateway` and `mm_network::configure_static_dns`.